Makefile.PL

```diff
@@ -0,0 +1,84 @@
+# This file was automatically generated by Dist::Zilla::Plugin::MakeMaker v6.015.
+use strict;
+use warnings;
+
+
+
+use ExtUtils::MakeMaker;
+
+my %WriteMakefileArgs = (
+  "ABSTRACT" => "Easy installation of the GSL library",
+  "AUTHOR" => "Joel A. Berger <joel.a.berger\@gmail.com>",
+  "BUILD_REQUIRES" => {
+    "Alien::Build" => "0.32",
+    "Alien::Build::MM" => "0.32",
+    "ExtUtils::MakeMaker" => "6.52"
+  },
+  "CONFIGURE_REQUIRES" => {
+    "Alien::Build" => "1.19",
+    "Alien::Build::MM" => "0.32",
+    "ExtUtils::MakeMaker" => "6.52"
+  },
+  "DISTNAME" => "Alien-GSL",
+  "LICENSE" => "perl",
+  "NAME" => "Alien::GSL",
+  "PREREQ_PM" => {
+    "Alien::Base" => "2.21",
+    "Sort::Versions" => 0
+  },
+  "TEST_REQUIRES" => {
+    "ExtUtils::MakeMaker" => "6.52",
+    "File::Spec" => 0,
+    "Test::More" => 0
+  },
+  "VERSION" => "1.01",
+  "test" => {
+    "TESTS" => "t/*.t"
+  }
+);
+
+
+my %FallbackPrereqs = (
+  "Alien::Base" => "2.21",
+  "Alien::Build" => "0.32",
+  "Alien::Build::MM" => "0.32",
+  "ExtUtils::MakeMaker" => "6.52",
+  "File::Spec" => 0,
+  "Sort::Versions" => 0,
+  "Test::More" => 0
+);
+
+
+# BEGIN code inserted by Dist::Zilla::Plugin::AlienBuild
+use Alien::Build::MM;
+my $abmm = Alien::Build::MM->new;
+%WriteMakefileArgs = $abmm->mm_args(%WriteMakefileArgs);
+# END code inserted by Dist::Zilla::Plugin::AlienBuild
+
+
+unless ( eval { ExtUtils::MakeMaker->VERSION(6.63_03) } ) {
+  delete $WriteMakefileArgs{TEST_REQUIRES};
+  delete $WriteMakefileArgs{BUILD_REQUIRES};
+  $WriteMakefileArgs{PREREQ_PM} = \%FallbackPrereqs;
+}
+
+delete $WriteMakefileArgs{CONFIGURE_REQUIRES}
+  unless eval { ExtUtils::MakeMaker->VERSION(6.52) };
```

Comment:
This fallback isn't necessary since we require 6.52 
https://github.com/PerlAlien/Alien-GSL/pull/8/files#diff-5d3ba18294715d9415e9e732852bfec6R15

Reply:
Updated with new commit. Also included GitHub action workflows that will run the tests on linux, macOS and Windows.